edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/ClauseFeatureExtractor.java

```diff
@@ -45,7 +45,14 @@ public ClauseFeatureExtractor(String pseudoParseViewName, String clauseViewName)
     @Override
     public Set<Feature> getFeatures(Constituent c) throws EdisonException {
         TextAnnotation ta = c.getTextAnnotation();
-        TreeView pseudoParseView = (TreeView) ta.getView(parseViewName);
+        TreeView pseudoParseView;
+        // If the real parse view exists, use that.
+        // Normally, this feature requires the PSEUDO_PARSE to be manually created by the user
+        // as it is cheaper to make than a full parse. However, there are cases
+        // where the user has already created a full parse view (for other features)
```

Comment:
> but in general it should be up to the user to decide whether they will add the PSEUDO_PARSE or the real parse  

Agreed; maybe add a TODO to make it parameterized? (i.e. the target view as parameter)